contracts/axelar-gateway/src/types.rs

```diff
@@ -25,9 +25,10 @@ pub enum ProofSignature {
     Unsigned,
 }
 
-/// `ProofSigner` represents a signer in a proof. If the signer submitted a signature,
-/// and if it is being included in the proof to meet the threshold, then a `ProofSignature` is attached.
-/// Otherwise, the `ProofSignature` is `Unsigned`.
+/// `ProofSigner` represents a signer in a proof.
+///
+/// If the signer submitted a signature, and if sit is being included in the proof to meet the threshold,
+/// then a `ProofSignature` is attached. Otherwise, the `ProofSignature` is `Unsigned`.
```

Comment:
```suggestion
/// If the signer submitted a signature, and if it is being included in the proof to meet the threshold,
/// then a signature is attached. Otherwise, the `ProofSignature` is `Unsigned`.
```

Reply:
nice catch, updated